### PR TITLE
Use familie-prosessering

### PR DESCRIPTION
**Pull request by MengZhuNAV**
Basically following things have been done:
1) use the familie-prosessering framework for tasks management
2) updated the flyway script to change some tables/data for familie-prosessering
3) some tests are using hibernate/H2, but familie-prosessering is using spring-jdbc, so they are patched to manually generate the tables for familie-prosessering

tested ok in preprod

### CODE REVIEW

**Review by tnarland — APPROVED** [review]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -23,6 +23,7 @@
         <kotlin.version>1.6.10</kotlin.version>
         <mockk.version>1.12.3</mockk.version>
         <confluent.version>6.2.1</confluent.version>
+        <prosessering.version>1.20211125150647_a7c6f64</prosessering.version>
```
Comment (stigebil) [inline pom.xml:26]: Burde teste med nyeste. 1.20220308102416_2479ea0

File: src/main/kotlin/no/nav/familie/ba/mottak/domene/Hendelseslogg.kt [hunk 2]
```diff
@@ -40,3 +53,33 @@ enum class HendelseConsumer {
         EF_VEDTAK_V1,
         EF_VEDTAK_INFOTRYGD_V1
 }
+
+/**
+ * JPA konverterer for å skrive ned en key=value text til et databasefelt (output tilsvarer java.util.Properties
+ * format).
+ */
+@Converter
+class PropertiesToStringConverter : AttributeConverter<Properties, String> {
```
Comment (stigebil) [inline src/main/kotlin/no/nav/familie/ba/mottak/domene/Hendelseslogg.kt:62]: Hører denne classen til Hendelselogg?

File: src/main/resources/db/migration/V10__prosessering_jdbc_patch.sql [hunk 3]
```diff
@@ -0,0 +1,7 @@
+ALTER TABLE task ALTER COLUMN id SET DEFAULT nextval('task_seq');
+ALTER SEQUENCE task_seq OWNED BY task.id;
+
+ALTER TABLE task_logg ALTER COLUMN id SET DEFAULT nextval('task_logg_seq');
+ALTER SEQUENCE task_logg_seq OWNED BY task_logg.id;
+
+UPDATE task SET versjon=1 WHERE versjon=0;
```
Comment (stigebil) [inline src/main/resources/db/migration/V10__prosessering_jdbc_patch.sql:7]: Just curious. Why do we need to update versjon?

**Review by stigebil — APPROVED** [review]
Great!

File: pom.xml (hunk 1 above)
Comment (MengZhuNAV) [reply pom.xml:26]: Ok

File: src/main/kotlin/no/nav/familie/ba/mottak/domene/Hendelseslogg.kt (hunk 2 above)
Comment (MengZhuNAV) [reply src/main/kotlin/no/nav/familie/ba/mottak/domene/Hendelseslogg.kt]: Moved away

**Pull request merged**